L/licensecheck/bundled/main.go

```diff
@@ -2,25 +2,26 @@ package main
 
 import "C"
 import (
-    "github.com/google/licensecheck"
+	"github.com/google/licensecheck"
 
-    "unsafe"
+	"unsafe"
 )
 
 //export License
 func License(msg *C.char) (**C.char, int, float64) {
-    bytes := []byte(C.GoString(msg))
-    cov := licensecheck.Scan(bytes)
+	bytes := []byte(C.GoString(msg))
+	cov := licensecheck.Scan(bytes)
 
-    // https://stackoverflow.com/a/41493208
-    cArray := C.malloc(C.size_t(len(cov.Match)) * C.size_t(unsafe.Sizeof(uintptr(0))))
-    a := (*[1<<30 - 1]*C.char)(cArray)
+	// https://stackoverflow.com/a/41493208
+	cArray := C.malloc(C.size_t(len(cov.Match)) * C.size_t(unsafe.Sizeof(uintptr(0))))
+	// mimic https://github.com/docker/docker-credential-helpers/pull/61
+	a := (*[(1 << 29) - 1]*C.char)(cArray)
```

Comment:
this is the only real diff; for some reason the file had tabs instead of spaces before, causing the huge diff here